rero_ils/modules/acq_receipts/api.py

```diff
@@ -196,6 +216,26 @@ def organisation_pid(self):
         """Shortcut for acquisition receipt organisation pid."""
         return extracted_data_from_ref(self.get('organisation'))
 
+    def get_receipt_lines(self, output=None):
+        """Get all receipt lines related to this receipt.
+
+        :param output: output method. 'count', 'pids' or None.
+        :return a generator of related order lines (or length).
+        """
+        def _list_object():
```

Comment:
What about moving the _list_object function outside the get_receipt_lines function as it is done here:
https://github.com/rero/rero-ils/pull/2522/commits/4235cdd0172c8e243da48d703d9f57d091076bd8#diff-6b9058a37fffba34f1924b8cc15325e1eae1871046e165421b99a447ce20896f
?
If you prefer you can leave it as it is and I could modify it in my PR 2522.